### ISSUE TITLE
taucdn.net

### ISSUE THREAD

**Issue by Marek33** [badmojr/1Hosts#13]
Taucdn.net is a CNAME domain, which is blocked on Firefox platforms, and it doesn't allow Revero.net to function, since I need to whitelist it. Without 1Hosts and Energized Ultimate @AdroitAdorKhan  , I only need to grey list it, and it works as it should.

**Comment by badmojr** [comment]
> Taucdn.net is a CNAME domain, which is blocked on Firefox platforms, and it doesn't allow Revero.net to function, since I need to whitelist it. Without 1Hosts and Energized Ultimate @AdroitAdorKhan , I only need to grey list it, and it works as it should.

Revero.net is currently parked.
It also does not use taucdn.net as a CNAME.
Pleae recheck...

**Comment by Marek33** [comment]
Why parked?

I did, and I still have that problem.

The logger says, that it connects to the **sa6gl.wpc.taucdn.net**, which Firefox warned me, because its apparently insecure, even if the website is 404.

**Comment by badmojr** [comment]
Please send a screenshot of the broken page..

**Comment by Marek33** [comment]
Here: ![image](https://user-images.githubusercontent.com/19204886/84908370-ac53db00-b0b4-11ea-93cd-ab5dea3befb8.png)

**Comment by Marek33** [comment]
Here: ![image](https://user-images.githubusercontent.com/19204886/84908528-dad1b600-b0b4-11ea-9732-5d7b3969a998.png)

**Comment by badmojr** [comment]
So, it’s reverso.net and not revero.net. 
Ok!

**Comment by badmojr** [comment]
The culprit was **cdn.reverso.net** .
See:- https://www.digwebinterface.com/?hostnames=cdn.reverso.net&type=&ns=resolver&useresolver=8.8.4.4&nameservers=

**Comment by badmojr** [comment]
Fixed!

**Comment by Marek33** [comment]
Yes, sorry, for the mistake, I was writing it quickly, and haven't noticed, that I have made a mistake.

Also, thanks for the fixes!
